### DISCUSSION TITLE
Game Stops working after choosing a stage.

### DISCUSSION THREAD

**Discussion by Visualwit** [Raytwo/ARCropolis#577] [Issues]
I have just updated my game to 13.0.2 and also updated ARCropolis and now when I choose a stage, the game stays at a black screen and the music stops playing. How can I fix this? Disabling all mods does not help.

**Comment by Julioiscool33** [comment]
Having this same, issue only happened after I updated as well /:

**Comment by Ditto30** [comment]
Delete and redownload everything on SD card (except the mods). This worked for me.

**Reply by Visualwit** [reply]
What is it you deleted? You don't mean the **ENTIRE** sd card?

**Reply by Ditto30** [reply]
I do mean the entire SD card unfortunately.

**Comment by Visualwit** [comment]
I fixed it. Some other plugins just needed updating.
